### PR TITLE
Bio-Tradis : addition of more labels for more clarity...

### PR DESCRIPTION
**Pull request by Delphine-L**
... and change of format to input embl and tab.gz

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by Delphine-L** [comment]
The test fails because it doesn't recognize the 'embl' input format. This should be resolved by the PR galaxyproject/galaxy#9293 .

**Comment by mvdbeek** [comment]
Yep, I'll finish that PR and come back to your PR. Can you bump the version numbers already ?

**Comment by Delphine-L** [comment]
By adding "+galaxy1" to the tool version?

**Comment by Delphine-L** [comment]
> Yep, I'll finish that PR and come back to your PR. Can you bump the version numbers already?

By the way, I used your PR to run the planemo tests with python 3.7, and all tests passed.

File: tool_collections/biotradis/bacteria_tradis.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="bacteria_tradis" name="Bio-TraDis reads to counts" version="@VERSION@">
+<tool id="bacteria_tradis" name="Bio-TraDis reads to counts" version="@TOOL_VERSION@+Galaxy@VERSION@">
```
Comment (mvdbeek) [inline tool_collections/biotradis/bacteria_tradis.xml]:
```suggestion
<tool id="bacteria_tradis" name="Bio-TraDis reads to counts" version="@TOOL_VERSION@+galaxy@VERSION@">
```

File: tool_collections/biotradis/tradis_essentiality.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<tool id="tradis_essentiality" name="Bio-TraDis Essentiality Predictions" version="@VERSION@">
+<tool id="tradis_essentiality" name="Bio-TraDis Essentiality Predictions" version="@TOOL_VERSION@+Galaxy@VERSION@">
```
Comment (mvdbeek) [inline tool_collections/biotradis/tradis_essentiality.xml]:
```suggestion
<tool id="tradis_essentiality" name="Bio-TraDis Essentiality Predictions" version="@TOOL_VERSION@+galaxy@VERSION@">
```

File: tool_collections/biotradis/tradis_gene_insert_sites.xml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<tool id="tradis_gene_insert_sites" name="Bio-TraDis counts to gene insertion data" version="@VERSION@">
+<tool id="tradis_gene_insert_sites" name="Bio-TraDis counts to gene insertion data" version="@TOOL_VERSION@+Galaxy@VERSION@">
```
Comment (mvdbeek) [inline tool_collections/biotradis/tradis_gene_insert_sites.xml]:
```suggestion
<tool id="tradis_gene_insert_sites" name="Bio-TraDis counts to gene insertion data" version="@TOOL_VERSION@+galaxy@VERSION@">
```

**Comment by mvdbeek** [comment]
Thanks @Delphine-L !

**Pull request merged**